### PR TITLE
IID-30 - update collection configuration

### PR DESCRIPTION
**Pull request by dariopelliccioli**
### List of changes

Update collection configuration for CosmosDB

### Motivation and context

Updated configuration to maintain aligned Terraform

### Type of changes

- [ ] Add new resources
- [X] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [X] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/idpay-common/03_database.tf [hunk 1]
```diff
@@ -75,4 +75,245 @@ module "mongdb_collection_onboarding_citizen" {
   ]
 
   lock_enable = false #TODO temporary only for dev
-}
\ No newline at end of file
+}
+
+# Collections
```
Comment (diegolagospagopa) [inline src/domains/idpay-common/03_database.tf:80]: Can we aggregate all this collections and have only one list to iterate?

Comment (diegolagospagopa) [inline src/domains/idpay-common/03_database.tf:80]: Please use a variable for the lock_enable so we can avoid errors in future and have a separate configuration for each env

**Review by diegolagospagopa — APPROVED** [review]

File: src/domains/idpay-common/03_database.tf (hunk 1 above)
Comment (antonioT90) [reply src/domains/idpay-common/03_database.tf]: As requested, it has been handled as a list

Comment (antonioT90) [reply src/domains/idpay-common/03_database.tf]: the lock now can be put to true on each environment

**Pull request merged**